lib/facility_factory.rb

```diff
@@ -0,0 +1,62 @@
+class FacilityFactory
+  attr_reader :new_facilities
+              :dmv_office_locations
+
+  def initialize
+    @new_facilities = []
```

Comment:
You need to test this. But would you even really need it if you created a method that returned an array of new Facility objects?